src/commands/game_commands/begin.ts

```diff
@@ -8,12 +8,15 @@ import { IPCLogger } from "../../logger";
 import MessageContext from "../../structures/message_context";
 import GameSession from "../../structures/game_session";
 import { state } from "../../kmq";
+import { competitionPrecheck } from "../../command_prechecks";
 
 const logger = new IPCLogger("begin");
 
 export default class BeginCommand implements BaseCommand {
+    preRunChecks = [{ checkFn: competitionPrecheck }];
+
     canStart(gameSession: GameSession, authorID: string, messageContext: MessageContext): boolean {
-        if (!gameSession || gameSession.gameType === GameType.CLASSIC) {
+        if (!gameSession || [GameType.CLASSIC, GameType.COMPETITION].includes(gameSession.gameType)) {
```

Comment:
Does it make more sense for this to be `gameSession.gameType !== GameType.TEAMS`

Reply:
I'd need to check that it's not elimination and it's not teams, but it does make more sense I think